src/ripple/protocol/impl/RippleAddress.cpp

```diff
@@ -706,16 +706,13 @@ std::string RippleAddress::humanGenerator () const
     }
 }
 
-void RippleAddress::setGenerator (Blob const& vPublic)
-{
-    mIsValid        = true;
-    SetData (VER_FAMILY_GENERATOR, vPublic);
-}
-
 RippleAddress RippleAddress::createGeneratorPublic (RippleAddress const& naSeed)
 {
     RippleAddress   naNew;
-    naNew.setGenerator (GenerateRootDeterministicPublicKey (naSeed.getSeed()));
+    
+    auto publicKey = GenerateRootDeterministicPublicKey (naSeed.getSeed());
```

Comment:
If `GenerateRootDeterministicPublicKey` is a function, it should start with a lower case letter...  not part of this change but just noted.